crossterm_input/README.md

```diff
@@ -41,7 +41,7 @@ When you want to use other modules as well you might want to use crossterm with
 
 ## Getting Started
 
-This documentation is only for `crossterm_input` version `0.3` if you have an older version I suggest you check the [Upgrade Manual](https://github.com/TimonPost/crossterm/blob/master/docs/UPGRADE.md). Also, check out the [examples](https://github.com/TimonPost/crossterm/tree/master/crossterm_input/examples) folders with detailed examples for all functionalities of this crate.
+This documentation is only for `crossterm_input` version `0.3` if you have an older version I suggest you check the [Upgrade Manual](https://github.com/crossterm-rs/crossterm/blob/master/docs/UPGRADE.md). Also, check out the [examples](https://github.com/crossterm-rs/crossterm/tree/master/crossterm_input/examples) folders with detailed examples for all functionalities of this crate.
```

Comment:
I'd suggest to remove following part of the sentence:

_This documentation is only for `crossterm_input` version `0.3`_

* Just to lower the maintenance burden
* Rewrite rest of the sentence in a way that have to consult upgrade manual if they're upgrading.